src/types/developer.ts

```diff
@@ -0,0 +1,8 @@
+export type DeveloperKey = 'Frontend' | 'Backend' | 'iOS';
+export type DeveloperType = `${DeveloperKey} Developer`;
```

Comment:
```typescript
} as const;
type DeveloperType = keyof typeof developers;
```
대신 DeveloperKey를 하드코딩 하신게 궁금..

`+`
```suggestion
export type DeveloperType = DevelperInfo['type'];
```

Reply:
아.. 이거 topics.ts 에 똑같은 이름으로 있어요!! 수정할게요
```
export const TOPIC_MAP = {
  Frontend: TOPICS_FRONTEND,
  Backend: TOPICS_BACKEND,
  iOS: TOPICS_IOS,
};

export type DeveloperKey = keyof typeof TOPIC_MAP;
```
+) DevelperInfo['type']; 이렇게 하면 순환한다고 에러나더라구요 그래서 처음에 하드코딩으로 넣었었어요,, ㅎㅎ,,

Reply:
아.. 그렇군여
이거 `DeveloperInfo`를 자세히 보면 사실은 `developers`의 구조를 설명하는데 `DeveloperType`은 `TOPIC_MAP` 에서 키를 가져와서 유지보수가 두배로 힘들어지고 구조적으로 이해도 어려워지는 느낌인데요..
합치는 거는 어려운가요?